util/flocks/src/tfrwlock/read_guard.rs

```diff
@@ -1,23 +1,28 @@
 use std::ops::Deref;
 use tokio::sync::RwLockReadGuard;
-use rustix::fd::AsFd;
-use crate::frwlock::{FrwLock, FrwLockReadGuard};
+use rustix::{
+    fs::{flock, FlockOperation},
+    fd::AsFd
+};
 
-pub struct TfrwLockReadGuard<'a, T: AsFd> {
-    _outer: RwLockReadGuard<'a, FrwLock<T>>,
-    inner: FrwLockReadGuard<T>,
+pub struct FileRwLockReadGuard<'a, T: AsFd> {
```

Comment:
As said elsewhere, I generally avoid lengthy prefixed names in favor of terser names disambiguated by module structure or `as` aliases if necessary: `use flocks::tokio::rwlock::ReadGuard as FileReadGuard;`